web/src/composebox_typeahead.ts

```diff
@@ -954,9 +955,17 @@ export function get_candidates(
                     topic_list.push(token);
                 }
                 const matcher = get_topic_matcher(token);
+                // We want the empty topic's display name to match the query
+                // while searching for topics in the typeahead.
+                const realm_empty_topic_name = util.get_final_topic_display_name("");
```

Comment:
Let's call it `empty_string_topic_display_name`.

We use the term `empty_topic` to refer to topics with no messages.